src/crypto/vss.rs

```diff
@@ -44,6 +47,34 @@ impl Vss {
             negative_secret: negative_secret,
         }
     }
+
+    pub fn create_node_shares(
+        index: usize,
+        private_key: &PrivateKey,
+        threshold: usize,
+        share_count: usize,
+    ) -> (VerifiableSS, Vec<FE>) {
+        assert!(
+            index > 0,
+            "index should be greater or equal to 1: {}",
+            index
+        );
+        assert!(
+            share_count >= threshold,
+            "share count should be greater or equal to threshold. share_count: {}, threshold: {}",
+            share_count,
+            threshold
+        );
+        assert!(
+            share_count >= index,
+            "index should be less or equal to share_count. share_count: {}, index: {}",
+            share_count,
+            index
+        );
+        let key = Sign::create_key(index, Sign::private_key_to_big_int(private_key.key));
```

Comment:
How about to create FE instance direct here like below.

```rust
 let secret: FE = ECScalar::from(&Sign::private_key_to_big_int(private_key.key.clone()).unwrap());
```

and put it into `VerifiableSS::share_at_indices` like:

```rust
VerifiableSS::share_at_indices(threshold - 1, share_count, &secret, &parties)
```

---
p.s. 

This is out of scope in this PR but I was just wondering.
I think `Sign::private_key_to_big_int` can return `BigInt` not `Option<BigInt>`. 
and it is useful to get the argument as a reference of SecretKey.